### ISSUE TITLE
Index import command should be able to import gzipped files

### ISSUE THREAD

**Issue by tautrimas** [ongr-io/ElasticsearchBundle#382]
Currently, importing is possible only on plain .json files. It should be very easy to enable gzipped file import by using this snippet:

```php
$fileHandler = @fopen(((substr($this->filename, -3) == '.gz') ? 'compress.zlib://' : '') . $this->filename, 'r');
```

https://github.com/ongr-io/ElasticsearchBundle/blob/e95531a69430f23bb886a39a221b449a69433f72/Service/Json/JsonReader.php#L99

**Comment by saimaz** [comment]
Any chance to get PR for this?

**Comment by ndinh215** [comment]
Do we need an option to distinguish between a normal file with a compressed Json file? For example, in MongoDB, they use the option '--jsonArray' to differently behave with traditional Json file and json objects array.
```
mongoimport --db <db-name> --collection <coll-name> --type json --file data.json
--jsonArray
```
How do you think? @saimaz @mvar @murnieza

**Comment by saimaz** [comment]
Yes you are absolutely correct. The provided example by cutting 3 last letters from the path is a wrong way. We should introduce option `--gzip` and then use `compress.zlib://` stream. If user messes up with the file extensions that his problem not our's.

**Comment by ndinh215** [comment]
OK, got it. Thanks @saimaz .

**Comment by ndinh215** [comment]
Closes #382.
